prow/cmd/autobump/bump.sh

```diff
@@ -137,7 +137,7 @@ list-options() {
 
 upstream-version() {
  local branch="https://raw.githubusercontent.com/kubernetes/test-infra/master"
- local file="prow/cluster/deck_deployment.yaml"
+ local file="config/prow/cluster/deck_deployment.yaml"
```

Comment:
We need to copy the `deck_deployment.yaml` file instead of moving it and keep the original file for a while to allow the old version of the autobump utility to bump itself to this version.

Reply:
A temporary symlink might be a better option.

Reply:
Does the autobump utility update the autobump job?

Reply:
Yes, it bumps itself and other jobs that use Prow component images like branchprotector, peribolos, checkconfig, etc. Example: https://github.com/istio/test-infra/blob/3a14b656a0578216498de2d5371b2948aaecb0f0/prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml#L97-L102